<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="abas">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tabV">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <attribute name="title">
          <string>Vendendo no WH</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="4">
           <widget class="QLineEdit" name="txt_nItensV">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="lb_nItens">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Total:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="7">
           <widget class="QTableWidget" name="tabela">
            <property name="rowCount">
             <number>0</number>
            </property>
            <property name="columnCount">
             <number>6</number>
            </property>
            <column>
             <property name="text">
              <string>%</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Lucro</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Item</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Pre√ßo Compra</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Estoque WH</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Link</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="botao_bp">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Check BP.TF</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="botao_sc">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Check SC.TF</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="lb_nEncontrados">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Encontrados:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLineEdit" name="txt_nEncontradosV">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabC">
         <attribute name="title">
          <string>Comprando no WH</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="5" column="7">
           <widget class="QCheckBox" name="chkMonitorar">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Monitorar</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QLineEdit" name="txt_nItensC">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="7">
           <widget class="QPushButton" name="btnNext">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>v</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Total:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="botao_escrow">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Check Escrow WH</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="8">
           <widget class="QTableWidget" name="tabelaCompra">
            <property name="rowCount">
             <number>0</number>
            </property>
            <property name="columnCount">
             <number>4</number>
            </property>
            <column>
             <property name="text">
              <string>Lucro</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Item</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Pre√ßo de Venda</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Estoque Atual</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="4" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="3" colspan="2">
           <widget class="QPushButton" name="botao_reservado">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>ITEM RESERVADO!</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="4" column="2" colspan="2">
         <widget class="QPushButton" name="botao_refresh">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOp_es">
    <property name="title">
     <string>Op√ß√µes Venda</string>
    </property>
    <widget class="QMenu" name="menuOrdena_o">
     <property name="title">
      <string>Ordena√ß√£o</string>
     </property>
     <addaction name="actionLucro"/>
     <addaction name="actionPercentual"/>
    </widget>
    <widget class="QMenu" name="menuTipo_Link">
     <property name="title">
      <string>Tipo Link</string>
     </property>
     <addaction name="actionOutpost"/>
     <addaction name="actionBP_Classifieds"/>
    </widget>
    <addaction name="menuOrdena_o"/>
    <addaction name="menuTipo_Link"/>
    <addaction name="actionConsiderar_Estoque"/>
   </widget>
   <widget class="QMenu" name="menuOp_es_Compra">
    <property name="title">
     <string>Op√ß√µes Compra</string>
    </property>
    <addaction name="actionConsiderar_Estoque_C"/>
   </widget>
   <widget class="QMenu" name="menuInforma_es">
    <property name="title">
     <string>Informa√ß√µes</string>
    </property>
    <addaction name="actionPrecos_Atuais"/>
   </widget>
   <addaction name="menuOp_es"/>
   <addaction name="menuOp_es_Compra"/>
   <addaction name="menuInforma_es"/>
  </widget>
  <action name="actionLucro">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Lucro</string>
   </property>
  </action>
  <action name="actionPercentual">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Percentual</string>
   </property>
  </action>
  <action name="actionOutpost">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Outpost</string>
   </property>
  </action>
  <action name="actionBP_Classifieds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BP Classifieds</string>
   </property>
  </action>
  <action name="actionConsiderar_Estoque">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Considerar Estoque</string>
   </property>
  </action>
  <action name="actionConsiderar_Estoque_C">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Considerar Estoque</string>
   </property>
  </action>
  <action name="actionPrecos_Atuais">
   <property name="text">
    <string>Pre√ßos Atuais</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
